pkgs/by-name/fv/fvm/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  buildDartApplication,
+  clang,
+  pkg-config,
+  gtk3,
+  cmake,
+  fetchFromGitHub,
+  nix-update-script,
+  lib,
+}:
+
+buildDartApplication rec {
+  pname = "fvm";
+  version = "3.2.1";
+
+  src = fetchFromGitHub {
+    owner = "leoafarias";
+    repo = pname;
+    tag = version;
+    hash = "sha256-i7sJRBrS5qyW8uGlx+zg+wDxsxgmolTMcikHyOzv3Bs=";
+  };
+
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+
+  nativeBuildInputs = [
+    clang
+    pkg-config
+    gtk3
+    cmake
+  ];
+
+  shellHook = ''
+    export PKG_CONFIG_PATH=${gtk3.dev}/lib/pkgconfig
+    export LD_LIBRARY_PATH=${gtk3.out}/lib:$LD_LIBRARY_PATH
```

Comment:
```suggestion
    export PKG_CONFIG_PATH=${lib.getDev gtk3}/lib/pkgconfig
    export LD_LIBRARY_PATH=${gtk3}/lib:$LD_LIBRARY_PATH
```

Reply:
This looks really strange though, are you sure the build doesn't work without this being set?